### PR TITLE
Start to convert browser/demo to manifest & planner.

### PR DESCRIPTION
**Pull request by dstockwell**

### CODE REVIEW

File: runtime/browser/lib/demo-base.js [hunk 1]
```diff
@@ -59,33 +56,25 @@ class DemoBase extends HTMLElement {
...
+    this.context.recipes = this.stage.recipes;
+    planner.init(this.arc, this.context);
+    let plans = await planner.plan(500);
+    plans.forEach((plan, i) => {
+      // TODO: invoke speculator/decriptinator
+      const Speculator = require('../../speculator.js');
+      const DescriptionGenerator = require('../../description-generator.js');
+      let rank = 1;
+      let description = plan.toString();
+      //let description = plan.particles.map(p => p.spec.name).join(', '); //new DescriptoinGenerator();
```
Comment (mmandlis) [inline runtime/browser/lib/demo-base.js]: none of the views in the plan have IDs - is it expected? the description generator relies on views having unique ids.

Comment (dstockwell) [inline runtime/browser/lib/demo-base.js]: That doesn't seem right, the recipe can't be resolved unless they are create or have an id: https://github.com/PolymerLabs/arcs/blob/master/runtime/recipe/view.js#L98

File: particles/Recommend/Recommend.js [hunk 2]
```diff
@@ -16,9 +16,12 @@ defineParticle(({Particle}) => {
         var populationView = views.get("population");
         this.logDebug("population", populationView);
         populationView.toList().then(data => {
-          views.get('recommendations').store(data[0]);
-          views.get('recommendations').store(data[1]);
-          views.get('recommendations').store(data[2]);
+          if (data.length > 0)
```
Comment (mmandlis) [inline particles/Recommend/Recommend.js]: can it be a for loop?

Comment (dstockwell) [inline particles/Recommend/Recommend.js]: Done!

**Pull request merged**